### PR TITLE
increase timeout for bucket policy unit tests

### PR DESCRIPTION
**Pull request by nadavMiz**
### Explain the changes
1. increase the timeout for unit tests that are failing on exceeded timeout.
2. timeout was increased to 15000, this number was chosen based on timeout that was set for similar tests in the same file 

### Issues: Fixed #xxx / Gap #xxx
1. 

### Testing Instructions:
1. run unit tests for bucket policy. tests should pass

- [ ] Doc added/updated
- [ ] Tests added

### CODE REVIEW

**Comment by nadavMiz** [comment]
some of the bucket policy sometimes fails in the CI for timeout exceeded. this is an example of the issue observed in the logs:

```
2023-09-26T15:46:31.1166445Z   3421 passing (23m)
2023-09-26T15:46:31.1177169Z   18 pending
2023-09-26T15:46:31.1188507Z   1 failing
2023-09-26T15:46:31.1199752Z
2023-09-26T15:46:31.1224495Z   1) s3_bucket_policy
2023-09-26T15:46:31.1224830Z        should be able to set bucket policy when none set:
2023-09-26T15:46:31.1225857Z      Error: Timeout of 2000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/root/node_modules/noobaa-core/src/test/unit_tests/index.js)
2023-09-26T15:46:31.1226417Z       at listOnTimeout (node:internal/timers:569:17)
2023-09-26T15:46:31.1226780Z       at process.processTimers (node:internal/timers:512:7)
```bash

File: src/test/unit_tests/test_s3_bucket_policy.js [hunk 1]
```diff
@@ -192,6 +192,8 @@ mocha.describe('s3_bucket_policy', function() {
     });
 
     mocha.it('should be able to set bucket policy when none set', async function() {
+        const self = this; // eslint-disable-line no-invalid-this
```
Comment (naveenpaul1) [inline src/test/unit_tests/test_s3_bucket_policy.js:195]: why we can not use this.timeout()? lint error?

**Review by shirady — APPROVED** [review]
LGTM

File: src/test/unit_tests/test_s3_bucket_policy.js (hunk 1 above)
Comment (nadavMiz) [reply src/test/unit_tests/test_s3_bucket_policy.js:195]: both ways cause lint errors. other tests in this file use ```const self = this``` so I did it this way to be consistent with the other tests

**Review by naveenpaul1 — APPROVED** [review]

**Pull request merged**